packages/uni_ui/lib/theme.dart

```diff
@@ -1,68 +1,86 @@
-const Color lightGray = Color.fromARGB(255, 229, 229, 229);
-const Color salmon = Color.fromARGB(255, 227, 145, 145);
+// Widget Colors
+const Color primaryVibrant = Color.fromARGB(255, 102, 9, 16);
+const Color secondary = Color.fromARGB(255, 255, 245, 243);
+const Color grayText = Color.fromARGB(255, 48, 48, 48);
+const Color grayMiddle = Color.fromARGB(255, 127, 127, 127);
+const Color background = Color.fromARGB(255, 255, 255, 255);
+const Color details = Color.fromARGB(235, 177, 77, 84);
+const Color divider = Color.fromARGB(255, 229, 229, 229);
 
-const _textTheme = TextTheme(
-  displayLarge: TextStyle(fontSize: 40, fontWeight: FontWeight.w400),
-  displayMedium: TextStyle(fontSize: 32, fontWeight: FontWeight.w400),
-  displaySmall: TextStyle(fontSize: 28, fontWeight: FontWeight.w400),
-  headlineLarge: TextStyle(fontSize: 28, fontWeight: FontWeight.w300),
-  headlineMedium: TextStyle(fontSize: 24, fontWeight: FontWeight.w300),
-  headlineSmall: TextStyle(fontSize: 20, fontWeight: FontWeight.w400),
-  titleLarge: TextStyle(fontSize: 18, fontWeight: FontWeight.w300),
-  titleMedium: TextStyle(fontSize: 17, fontWeight: FontWeight.w300),
-  titleSmall: TextStyle(fontSize: 16, fontWeight: FontWeight.w300),
-  bodyLarge: TextStyle(fontSize: 16, fontWeight: FontWeight.w400),
-  bodyMedium: TextStyle(fontSize: 14, fontWeight: FontWeight.w400),
-  bodySmall: TextStyle(fontSize: 13, fontWeight: FontWeight.w400),
-);
+const _lightTextTheme = TextTheme(
+    displayLarge: TextStyle(
+        fontSize: 40, fontWeight: FontWeight.w400, color: primaryVibrant),
+    displayMedium:
+        TextStyle(fontSize: 40, fontWeight: FontWeight.w400, color: grayText),
+    displaySmall:
+        TextStyle(fontSize: 28, fontWeight: FontWeight.w400, color: grayText),
+    headlineLarge:
+        TextStyle(fontSize: 20, fontWeight: FontWeight.w300, color: grayText),
+    headlineMedium: TextStyle(
+        fontSize: 20, fontWeight: FontWeight.w300, color: primaryVibrant),
+    headlineSmall: TextStyle(
+        fontSize: 16, fontWeight: FontWeight.w400, color: primaryVibrant),
+    titleLarge:
+        TextStyle(fontSize: 16, fontWeight: FontWeight.w300, color: background),
+    titleMedium:
+        TextStyle(fontSize: 12, fontWeight: FontWeight.w300, color: grayText),
+    titleSmall:
+        TextStyle(fontSize: 12, fontWeight: FontWeight.w300, color: background),
+    bodyLarge:
+        TextStyle(fontSize: 12, fontWeight: FontWeight.w400, color: grayText),
+    bodyMedium:
+        TextStyle(fontSize: 12, fontWeight: FontWeight.w400, color: grayMiddle),
+    bodySmall: TextStyle(
+        fontSize: 12, fontWeight: FontWeight.w400, color: primaryVibrant),
+    labelLarge:
+        TextStyle(fontSize: 9, fontWeight: FontWeight.w400, color: grayText),
+    labelMedium:
+        TextStyle(fontSize: 9, fontWeight: FontWeight.w400, color: grayMiddle),
+    labelSmall: TextStyle(
+        fontSize: 9, fontWeight: FontWeight.w400, color: primaryVibrant));
```

Comment:
Well done! 
The size and colors are all correct, but it seems the fontWeight isn't according to figma in displaySmall, headlines and tittles.